website/common/locales/en/npc.json

```diff
@@ -61,7 +61,7 @@
     "paymentMethods": "Purchase using",
 
     "classGear": "Class Gear",
-    "classGearText": "First: don't panic! Your old gear is in your inventory, and you're now wearing the apprentice equipment of your new class. Wearing your class's gear grants you a 50% bonus to stats. However, feel free to switch back to your old gear.",
+    "classGearText": "Congratulations on choosing a class! I've gone ahead and added your new basic weapon to your inventory. Take a look below to equip it!",
```

Comment:
When a new english string is uploaded to Transifex all old translations are removed, so there should be no problems